tests/test_graphite_send.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from aiographite.aiographitesend import AioGraphiteSendException
+from aiographite.aiographitesend import PlaintextProtocol, PickleProtocol, AIOGraphite
+import socket
+import unittest
+import asyncio
+from asyncio import test_utils
+
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+
+
+
+def setup_module(module):
+	module.server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+	module.server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+	module.server.bind(('localhost', DEFAULT_GRAPHITE_PLAINTEXT_PORT))
+	module.server.listen(20)
+
+
+def teardown_module(module):
+	"""
+		1. Destroy aiographite_send instance
+		2. Close the server socket
+	"""
+	try:
+		module.server.shutdown(socket.SHUT_RD)
+		module.server.close()
+	except Exception:
+		pass
+	module.server = None
+
+
+
+@pytest.fixture
+def metric_dir_list():
```

Comment:
are these fixtures for later use? you need to put them in conftest.py file and then you can use them in your unit tests .